future-apps/payroll/contracts/Payroll.sol

```diff
@@ -612,25 +604,33 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
         emit TerminateEmployee(_employeeId, employee.accountAddress, _endDate);
     }
 
-    function _getOwedSalary(uint256 _employeeId, uint64 _date) internal view returns (uint256) {
+    function _getLastPayroll(uint256 _employeeId, uint256 _payedAmount) internal view returns (uint64) {
+        Employee storage employee = employees[_employeeId];
+        uint64 timeDiff = uint64(_payedAmount.div(employee.denominationTokenSalary));
```

Comment:
Sorry, I meant `Uint256Helpers.toUint64`

Reply:
Discussed offline, we were referring to use `Uint256Helpers` actually

Reply:
Going over it again, I see this is only called from `_payday`, so `_payedAmount` will always be <= `owedAmount`, which comes from `_getOwedSalary`, which does the opposite calculation but with a date `<= getTimestamp64()`, so this can not ever be greater than the  current date, which is fine. Anyway, as a sanity check, I would still use that helper, or, even better, check than the result is not greater than `getTimestamp64()`, as it doesn't seem to make sense otherwise.